kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -1969,6 +1972,50 @@ bool IndexerASTVisitor::VisitInitListExpr(const clang::InitListExpr* ILE) {
   return true;
 }
 
+bool IndexerASTVisitor::TraverseCallExpr(clang::CallExpr* CE) {
+  auto callee = CE->getDirectCallee();
+  auto callee_exp = CE->getCallee();
+  bool valid = callee != nullptr && callee_exp != nullptr &&
+               callee->param_size() == CE->getNumArgs();
```

Comment:
So this doesn't work for functions with default arguments? or are default arguments included in getNumArgs()?

Please document when these conditions might fail, if you know.

Reply:
I was a little too conservative with this check. I've also added tests showing that this doesn't fail (in the crashy sense) with varargs or pack expansions.